source/chapters/25_sprites_and_walls/step_5.py

```diff
@@ -48,28 +48,28 @@ def setup(self):
         self.score = 0
 
         # Create the player
-        self.player_sprite = arcade.Sprite("images/character.png", SPRITE_SCALING_PLAYER)
+        self.player_sprite = arcade.Sprite(":resources:images/animated_characters/female_adventurer/femaleAdventurer_idle.png", SPRITE_SCALING_PLAYER)
```

Comment:
The idea for this example was to get students to practice downloading and using their own images, rather than  the build-in resources.